src/main/src/main/java/org/geoserver/config/RequestLoggingInfo.java

```diff
@@ -0,0 +1,21 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.config;
+
+import org.geoserver.catalog.Info;
+
+public interface RequestLoggingInfo extends Info {
```

Comment:
Okay, so this is a public class and will need javadocs - but more importantly it is a change to configuration on disk and will need a change proposal.

For starts I would double check if you can add the extra isRequestLoggingEnabled and isRequestBodiesEnabled to global settings, there is a metadata map which is ... specifically done so you can add extra settings like this without having to make a change proposal.

Reply:
Thanks Jody.  It would make more sense to piggyback off of an existing settings model.  When you refer to a "metadata map" are you talking about the MetadataMap called "metadata" within GeoServerInfoImpl?

Reply:
Or are you referring in general to the globalInfoModel on the GlobalSettingsPage?

Reply:
I that sounds about right

Reply:
Fixed to use MetadataMap